js/app/modules/card/audio.js

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017 Endless Mobile, Inc.
+
+const GObject = imports.gi.GObject;
+const Gtk = imports.gi.Gtk;
+
+const ArticleContent = imports.app.interfaces.articleContent;
+const Card = imports.app.interfaces.card;
+const EosKnowledgePrivate = imports.gi.EosKnowledgePrivate;
+const Module = imports.app.interfaces.module;
+
+// Make sure included for glade template
+const Utils = imports.app.utils;
+
+/**
+ * Class: Audio
```

Comment:
Going forward, we really need better documentation at the time we introduce new modules, because one thing that will never happen is going back and making a comprehensive review of documentation. Could you please make sure that this comment and the property below make sense as part of the modular framework API documentation?